[DOC] pulsar web service executor metrics doc (#14745)
apache/pulsar: site2/docs/reference-metrics.md

| jetty_stats_seconds | Gauge | Time in seconds stats have been collected for. |
| jetty_responses_bytes_total | Counter | Total number of bytes across all responses. |

### Web service executor metrics

> > For a functions-worker running separately from brokers, its Jetty metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.

Reviewer: ```suggestion
> For functions workers running separately from brokers, their Jetty metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.
```

All the web service executor metrics are labelled with the following labels:

- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.

| Name | Type | Description |
|---|---|---|
| pulsar_web_executor_max_threads | GAUGE | The max threads of pulsar-web  thread pool |

> | pulsar_web_executor_min_threads | GAUGE | The max threads of pulsar-web thread pool |

Reviewer: the min threads

| pulsar_web_executor_idle_threads | GAUGE | The idle threads of pulsar-web thread pool |

> | pulsar_web_executor_active_threads | GAUGE | How many threads doing task of pulsar-web thread pool |

Reviewer: ```suggestion
| pulsar_web_executor_active_threads | GAUGE | The number of threads performing tasks of pulsar-web thread pool |
```

> | pulsar_web_executor_current_threads | GAUGE | How many threads in the pulsar-web thread pool now |

Reviewer: ```suggestion
| pulsar_web_executor_current_threads | GAUGE | The number of threads in the pulsar-web thread pool |
```
